### PR TITLE
Move registerProtocol method from preload, remove empty preload

### PR DESCRIPTION
**Pull request by kunstefix**

### CODE REVIEW

**Comment by LinFeiLong** [comment]
Interesting, what is it doing ?

**Comment by kunstefix** [comment]
> Interesting, what is it doing ?

Solving story [4334](https://app.clubhouse.io/mycrypto/story/4334/cannot-read-property-registerschemesasprivileged-of-undefined)

**Comment by LinFeiLong** [comment]
Do you feel it needs to be replaced ?
Or we just need to get rid of it ?

**Comment by kunstefix** [comment]
> Do you feel it needs to be replaced ?
> Or we just need to get rid of it ?

The fix is to move method call out of preload. After that the preload is empty so it can be removed.

**Comment by LinFeiLong** [comment]
Ok perfect

File: electron-app/main/index.ts [hunk 1]
```diff
@@ -20,7 +23,7 @@ app.on('activate', () => {
 
 // Create main BrowserWindow when electron is ready
 app.on('ready', () => {
-  getWindow();
+  getWindow().webContents.openDevTools();
```
Comment (LinFeiLong) [inline electron-app/main/index.ts]: I don't mind keeping it for the moment.
I'll add a comment though.

**Pull request merged**